sonic_installer/main.py

```diff
@@ -530,10 +532,17 @@ def install(url, force, skip_migration=False, skip_package_migration=False,
             echo_and_log('Error: Failed to set image as default', LOG_ERR)
             raise click.Abort()
     else:
+        # Verify that the binary image is secureboot
```

Comment:
>Verify that the binary image is secureboot [](http://example.com/codeflow?start=10&length=42)

Not accurate.
"Verify not installing non-secure image in a secure running image"

Reply:
Should we leave the echo message as it previous states?
"Image file '{}' is of a different type than running image"

Reply:
yes, we can keep the ech omessage.